apps/admin-x-activitypub/src/hooks/use-activity-pub-queries.ts

```diff
@@ -1152,14 +1139,20 @@ export function useNoteMutationForUser(handle: string, actorProps?: ActorPropert
     });
 }
 
-export function useAccountForUser(handle: string) {
+export function useAccountForUser(handle: string, profileHandle: string) {
+    const queryClient = useQueryClient();
+
+    // Invalidate the cache when profileHandle changes
+    React.useEffect(() => {
+        queryClient.invalidateQueries({queryKey: QUERY_KEYS.account(profileHandle)});
+    }, [profileHandle, queryClient]);
```

Comment:
This seems 🐟 but I can't reason about why. Why do we need this?

Reply:
You were right at pointing this out. Not needed. Removed it. 

I have also updated the cache to use `QUERY-Key.account` in place of `QUERY-Key.profile`, as we have removed the hook updating `QUERY-Key.profile`